src/index.js

```diff
@@ -63,7 +65,13 @@ function run(
       }
     }
     const schemaPath = args._[0]
-    loadSchemaJSON(schemaPath).then(schema => {
+    const headers = [].concat(args['header'] || []).reduce((obj, header) => {
+      const [key, value] = String(header).split('=', 2)
```

Comment:
Not sure if `=` in headers need to be escaped anyway, but let's do this safer – 2nd arg to `split` notably still splits the string on all `=` but only returns 2 results. If the arg was `Foo=x=y` this would return `Foo, x` but should be `Foo, x=y`.